sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/api/Azure.ResourceManager.RecoveryServicesBackup.netstandard2.0.cs

```diff
@@ -465,6 +465,8 @@ public static partial class RecoveryServicesBackupExtensions
         public static Azure.Response<Azure.ResourceManager.RecoveryServicesBackup.ProtectionContainerResource> GetProtectionContainerResource(this Azure.ResourceManager.Resources.ResourceGroupResource resourceGroupResource, string vaultName, string fabricName, string containerName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.RecoveryServicesBackup.ProtectionContainerResource>> GetProtectionContainerResourceAsync(this Azure.ResourceManager.Resources.ResourceGroupResource resourceGroupResource, string vaultName, string fabricName, string containerName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public static Azure.ResourceManager.RecoveryServicesBackup.ProtectionContainerResourceCollection GetProtectionContainerResources(this Azure.ResourceManager.Resources.ResourceGroupResource resourceGroupResource) { throw null; }
+        public static Azure.Pageable<Azure.ResourceManager.RecoveryServicesBackup.ProtectionContainerResource> GetProtectionContainerResourcesByBackupDeletedProtectionContainer(this Azure.ResourceManager.Resources.ResourceGroupResource resourceGroupResource, string vaultName, string filter = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
According to the doc in spec, this method should be renamed to `GetSoftDeletedProtectionContainers` by using

override-operation-name:
  DeletedProtectionContainers_List: GetSoftDeletedProtectionContainers

Reply:
Sure @ArthurMa1978, will make this change and update shortly. could you pls point me to the doc in spec

Reply:
@ArthurMa1978 made the change as suggested